workitem/workitem_repository.go

```diff
@@ -600,7 +600,29 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, spaceID uuid.UUID,
 	if !wiType.CanConstruct {
 		return nil, errors.NewForbiddenError(fmt.Sprintf("cannot construct work items from \"%s\" (%s)", wiType.Name, wiType.ID))
 	}
-
+	var exists bool
+	// Prohibit creation of work items from a type that doesn't belong to current space template
+	query := fmt.Sprintf(`
+		SELECT EXISTS (
+			SELECT 1 from %s where id::text = '%s' AND space_template_id = (
+				SELECT space_template_id from spaces where id::text = '%s'
+			)
+		)`, wiType.TableName(), wiType.ID.String(), spaceID.String())
+	r.db.LogMode(true)
```

Comment:
This should be removed.

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/2115/commits/07bdc99b03dd6bf34a63ba53ebf63048ccb2d5ac